keyboards/mechlovin/infinity875/config.h

```diff
@@ -0,0 +1,91 @@
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID   0x4D4C // ML
+#define PRODUCT_ID  0x0875
+#define DEVICE_VER  0x0001
+#define PRODUCT     Infinity87.5
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 17
+ 
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D5, D2, D4, D3, D0, D1 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION ROW2COL
+
+#define LED_CAPS_LOCK_PIN D6
+#define LED_SCROLL_LOCK_PIN B4
+#define LED_PIN_ON_STATE 1
+
+#ifdef RGB_MATRIX_ENABLE
+#define RGB_DI_PIN E2
+#define DRIVER_LED_TOTAL 42
+#define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
+#define RGB_MATRIX_LED_FLUSH_LIMIT 16 // limits in milliseconds how frequently an animation will update the LEDs. 16 (16ms) is equivalent to limiting to 60fps (increases keyboard responsiveness)
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 220 // limits maximum brightness of LEDs to 200 out of 255. If not defined maximum brightness is set to 255
+#define VIA_QMK_RGBLIGHT_ENABLE
+#define DISABLE_RGB_MATRIX_ALPHAS_MODS	
+#define DISABLE_RGB_MATRIX_GRADIENT_UP_DOWN	
+#define DISABLE_RGB_MATRIX_GRADIENT_LEFT_RIGHT	
+#define DISABLE_RGB_MATRIX_BREATHING	
+#define DISABLE_RGB_MATRIX_BAND_SAT	
+#define DISABLE_RGB_MATRIX_BAND_VAL	
+#define DISABLE_RGB_MATRIX_BAND_PINWHEEL_SAT	
+#define DISABLE_RGB_MATRIX_BAND_PINWHEEL_VAL	
+#define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT	
+#define DISABLE_RGB_MATRIX_BAND_SPIRAL_VAL	
+#define DISABLE_RGB_MATRIX_CYCLE_ALL	
+#define DISABLE_RGB_MATRIX_CYCLE_LEFT_RIGHT	
+#define DISABLE_RGB_MATRIX_CYCLE_UP_DOWN	
+#define DISABLE_RGB_MATRIX_RAINBOW_MOVING_CHEVRON	
+#define DISABLE_RGB_MATRIX_CYCLE_OUT_IN	
+#define DISABLE_RGB_MATRIX_CYCLE_OUT_IN_DUAL	
+#define DISABLE_RGB_MATRIX_CYCLE_PINWHEEL	
+#define DISABLE_RGB_MATRIX_CYCLE_SPIRAL	
+#define DISABLE_RGB_MATRIX_DUAL_BEACON	
+#define DISABLE_RGB_MATRIX_RAINBOW_BEACON	
+#define DISABLE_RGB_MATRIX_RAINBOW_PINWHEELS	
+#define DISABLE_RGB_MATRIX_RAINDROPS	
+#define DISABLE_RGB_MATRIX_JELLYBEAN_RAINDROPS	
+#define DISABLE_RGB_MATRIX_HUE_BREATHING	
+#define DISABLE_RGB_MATRIX_HUE_PENDULUM	
+#define DISABLE_RGB_MATRIX_HUE_WAVE	
```

Comment:
RGB Matrix requires explicit enabling of RGB Modes now. 

https://docs.qmk.fm/#/feature_rgb_matrix?id=rgb-matrix-effects

```suggestion
#define ENABLE_RGB_MATRIX_ALPHAS_MODS
#define ENABLE_RGB_MATRIX_GRADIENT_UP_DOWN
#define ENABLE_RGB_MATRIX_GRADIENT_LEFT_RIGHT
#define ENABLE_RGB_MATRIX_BREATHING
#define ENABLE_RGB_MATRIX_BAND_SAT
#define ENABLE_RGB_MATRIX_BAND_VAL
#define ENABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
#define ENABLE_RGB_MATRIX_BAND_PINWHEEL_VAL
#define ENABLE_RGB_MATRIX_BAND_SPIRAL_SAT
#define ENABLE_RGB_MATRIX_BAND_SPIRAL_VAL
#define ENABLE_RGB_MATRIX_CYCLE_ALL
#define ENABLE_RGB_MATRIX_CYCLE_LEFT_RIGHT
#define ENABLE_RGB_MATRIX_CYCLE_UP_DOWN
#define ENABLE_RGB_MATRIX_RAINBOW_MOVING_CHEVRON
#define ENABLE_RGB_MATRIX_CYCLE_OUT_IN
#define ENABLE_RGB_MATRIX_CYCLE_OUT_IN_DUAL
#define ENABLE_RGB_MATRIX_CYCLE_PINWHEEL
#define ENABLE_RGB_MATRIX_CYCLE_SPIRAL
#define ENABLE_RGB_MATRIX_DUAL_BEACON
#define ENABLE_RGB_MATRIX_RAINBOW_BEACON
#define ENABLE_RGB_MATRIX_RAINBOW_PINWHEELS
#define ENABLE_RGB_MATRIX_RAINDROPS
#define ENABLE_RGB_MATRIX_JELLYBEAN_RAINDROPS
#define ENABLE_RGB_MATRIX_HUE_BREATHING
#define ENABLE_RGB_MATRIX_HUE_PENDULUM
#define ENABLE_RGB_MATRIX_HUE_WAVE
#define ENABLE_RGB_MATRIX_PIXEL_FRACTAL
#define ENABLE_RGB_MATRIX_PIXEL_RAIN

#define ENABLE_RGB_MATRIX_TYPING_HEATMAP
#define ENABLE_RGB_MATRIX_DIGITAL_RAIN

#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTIWIDE
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTICROSS
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTINEXUS
#define ENABLE_RGB_MATRIX_SPLASH
#define ENABLE_RGB_MATRIX_MULTISPLASH
#define ENABLE_RGB_MATRIX_SOLID_SPLASH
#define ENABLE_RGB_MATRIX_SOLID_MULTISPLASH
```